PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+Thanks for contributing to Apache Flink, before you open your PR please kindly take into consideration the following check list.
+Once you are sure, all items on the list can be checked, feel free to open your PR. For more information and/or questions please refer to the [How To Contribute guide](http://flink.apache.org/how-to-contribute.html).
```

Comment:
Suggestion: If your changes take all of the items into account, feel free to open your pull request.